# Define the goal
std_msgs/Float64[] centreOfFrame   
---
# Define the result
std_msgs/Bool Reached
---
# Define a feedback message
std_msgs/Float64[] currentCoordinates